.github/workflows/pull-request.yml

```
@@ -114,3 +116,34 @@ jobs:
       - name: Build kitty-krew app
         working-directory: apps/kitty-krew
         run: bun run build
+
+  reviseur:
+    needs: check_changed_files
+    if: ${{ needs.check_changed_files.outputs.reviseur == 'true' }}
```

actionlint (1.7.4)

property "reviseur" is not defined in object type {findbobastore: string; kitty-krew: string; proompteng: string}

(expression)